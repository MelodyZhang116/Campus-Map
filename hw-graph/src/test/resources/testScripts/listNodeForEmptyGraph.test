# test that whether the node in empty graph can be listed correctly.

# Create a graph
CreateGraph graph1

# Print the nodes in the graph
ListNodes graph1
