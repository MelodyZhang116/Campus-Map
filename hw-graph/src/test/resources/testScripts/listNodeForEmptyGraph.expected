# test that whether the node in empty graph can be listed correctly.

# Create a graph
created graph graph1

# Print the nodes in the graph
graph1 contains: 